CMSgov/design-system
# [WNMGDS-855] Create github search utility

## Summary
The purpose of this PR is to create a utility we can use to search the Github API for which projects are using which design system versions.

## How to test
In the top right of CMS github, click your user icon > Settings.
[Choose Developer Settings in the left menu](https://github.com/settings/apps).
Choose personal access tokens.
Create a personal access token and write down the token.

- Pull down this branch
- cd to the utilities/findDesignVersion folder
- yarn install
- From the command line run `ts-node findDesignVersions.ts -- YOUR_PERSONAL_ACCESS_TOKEN`

It should print out repositories, file names, and design system versions.

Some of the respositories though match the search term, but don't have version numbers, example: 
![image](https://user-images.githubusercontent.com/21293083/113589662-a3aaa200-95ff-11eb-8883-f03750ae4c2d.png)


Should we consider completely ignoring some repositories?
